jenkins/opensearch/Jenkinsfile

```diff
@@ -132,6 +132,21 @@ pipeline {
                 success {
```

Comment:
I think this should have been a pipeline step as opposed to something that executes upon a successful build. We are also publishing a success notification prematurely, the docker build can fail, but we wouldn't know in Slack.

Reply:
If needed we can block the bundle build completion until docker build completes and return status.